website/docs/reference/appsmith-framework/widget-actions/post-message.md

```diff
@@ -0,0 +1,77 @@
+# Post Message
+
+The **Post message** action is used to enable safe cross-origin communication between [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window) objects; for example, between a page and a pop-up that it has created, or between a page and an iframe embedded within it. The Appsmith function `postWindowMessage` works by using the JavaScript function [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) so that you can communicate between applications in situations such as:
```

Comment:
I made some changes to the suggestion above, I this sounds clearer and shorter than my original :)

> The **Post message** action uses the `postWindowMessage()` function to enable safe cross-origin communication between different [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window) objects; for example, between an app and an iframe embedded within it. This may be useful in situations like: